explore/src/main/scala/explore/observationtree/TargetObsList.scala

```diff
@@ -760,7 +779,7 @@ object TargetObsList {
                               <.span(ExploreStyles.ObsTreeGroupHeader)(
                                 <.span(
                                   opIcon,
-                                  asterism.name.value,
+                                  asterism.name.map(_.value),
```

Comment:
Would this show `Some(name)` on the UI?

Reply:
I think it will just show `name` and skip it if it's `None`, as per `scalajs-react`'s conversion of an `Option[String]` into `VdomNode`.